src/main/java/org/owasp/benchmark/score/BenchmarkScore.java

```diff
@@ -653,263 +653,14 @@ private static TestResults readActualResults(File fileToParse) throws Exception
 		String filename = fileToParse.getName();
 		TestResults tr = null;
 
-		if ( filename.endsWith( ".csv" ) ) {
```

Comment:
You can't just rip out support for a bunch of other tools when changing/improving the Contrast tool support.